src/main/scala/firrtl/passes/Inline.scala

```diff
@@ -353,8 +358,11 @@ class InlineInstances extends Transform with DependencyAPIMigration with Registe
         Some(m.map(onStmt(ModuleName(m.name, CircuitName(c.main)))))
     })
 
-    val renames = renamesSeq.tail.foldLeft(renamesSeq.head)(_ andThen _)
+    val renames = renamesSeq match {
```

Comment:
```suggestion
    val renames = renamesSeq.reduceLeftOption(_ andThen _)
```

Reply:
This seems clean to me.

Reply:
(suggestion should also delete following lines)

Reply:
Haha yeah, I basically assume nobody will ever just click-accept them.

Reply:
Updated in the PR on this PR.